### PR TITLE
Update Multiplayer.xml

### PR DESCRIPTION
**Pull request by ghost**
Hi, i'm kinda new here but I wanted to help translate this mod.
Dutch is my native language so I didn't use google translate.

I fixed some minor grammar and spelling errors.

### CODE REVIEW

File: Languages/Dutch/Keyed/Multiplayer.xml [hunk 1]
```diff
@@ -25,18 +25,18 @@
...
-  <MpChangeUsernameInfo>U kunt uw gebruikersnaam wijzigen in de wijzigingsinstellingen.</MpChangeUsernameInfo>
+  <MpChangeUsernameInfo>U kunt uw gebruikersnaam wijzigen in de mod instellingen.</MpChangeUsernameInfo>
   
   <MpPlayerConnected>{0} heeft zich aangesloten</MpPlayerConnected>
   <MpPlayerDisconnected>{0} is vertrokken</MpPlayerDisconnected>
   
   <MpReplayOutdated>"Niet up-to-date"</MpReplayOutdated>
-  <MpReplayOutdatedDesc1>Deze herhaling is gemaakt voor de protocolversie {0}, maar je bent bezig met de versie die je gebruikt... {1}.</MpReplayOutdatedDesc1>
-  <MpReplayOutdatedDesc2>De herhalingen worden mogelijks niet correct afgespeeld.</MpReplayOutdatedDesc2>
+  <MpReplayOutdatedDesc1>Deze herhaling is gemaakt voor de protocolversie {0}, maar jij gebruikt versie {1}.</MpReplayOutdatedDesc1>
```
Comment (MilanDierick) [inline Languages/Dutch/Keyed/Multiplayer.xml:34]: "u" i.p.v. "jij"

**Review by MilanDierick — CHANGES_REQUESTED** [review]

File: Languages/Dutch/Keyed/Multiplayer.xml (hunk 1 above)
Comment (tindolt-zz) [reply Languages/Dutch/Keyed/Multiplayer.xml:34]: I think you had a stroke bud

Comment (SweptWasTaken) [reply Languages/Dutch/Keyed/Multiplayer.xml:34]: Does he just want to change MpReplayOutDatedDesc1? or Change it and MpReplayOutDatedDesc2? I'm assuming its a mistype.

Comment (tindolt-zz) [reply Languages/Dutch/Keyed/Multiplayer.xml:34]: Not sure we need him to respond first.

Comment (MilanDierick) [reply Languages/Dutch/Keyed/Multiplayer.xml:34]: My requested changes still need to be applied, the way it is formulated right now is the "impolite way", and it needs to be written in the "polite way" (I don't know how it's called in English).

**Comment by MilanDierick** [comment]
@heicomacje Please take a look at the requested change.

**Comment by MilanDierick** [comment]
@heicomacje Please take a look at the requested change.

**Comment by MilanDierick** [comment]
Closing this because these changes have been pushed in another PR: https://github.com/Parexy/Multiplayer/pull/65

**Pull request closed without merging**